tower-http/src/services/fs/serve_dir.rs

```diff
@@ -51,6 +53,15 @@ impl ServeDir {
         self.append_index_html_on_directories = append;
         self
     }
+    /// Cache-Control:max-age=300.
+    ///
+    /// This speeds up user's feeling for static sites, especially large files.
+    ///
+    /// Defaults to 300 seconds.
```

Comment:
I think these docs should be phrased differently. What will setting this actually make the browser do?